### PR TITLE
New viaggiatreno sensor documentation

### PR DESCRIPTION
**Pull request by fabfurnari**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

File: source/_components/sensor.viaggiatreno.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+description: "Check Italian public train delays from Viaggiatreno"
+date: 2016-10-25 16:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Transport
+logo: db.png
+ha_iot_class: "Cloud Polling"
+ha_release: 0.31
```
Comment (fabaff) [inline source/_components/sensor.viaggiatreno.markdown]: Will be 0.32, as 0.31 was released a some days ago.

File: source/_components/sensor.viaggiatreno.markdown [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+The `viaggiatreno` sensor can be used to check specific train delay at certain stop. It needs the
+train number as well as the station name (as displayed in http://viaggiatreno.it)
```
Comment (fabaff) [inline source/_components/sensor.viaggiatreno.markdown]: No line breaks needed.

File: source/_components/sensor.viaggiatreno.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  - platform: viaggiatreno
+    station_name: ROMA TERMINI
+    train_no: 34028
+```
+
+Configuration variables:
+
+- **station_name** (*Required*): The name of the station where delay is calculated.
+- **train_no** (*Required*): Train number as displayed in http://viaggiatreno.it
+- **origin_station** (*Optional*): Departure station ID (if not provided can be calculated)
```
Comment (fabaff) [inline source/_components/sensor.viaggiatreno.markdown]: Perhaps a little note about how this station can be calculated would be nice to help the users.

File: source/_components/sensor.viaggiatreno.markdown [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The sensor provides (at the moment) just the delay expressed in minutes. If the specified train
+has already passed the station it returns None
```
Comment (fabaff) [inline source/_components/sensor.viaggiatreno.markdown]: Missing period.

**Comment by fabfurnari** [comment]
Thanks for the feedback, should be fixed now

**Pull request closed without merging**